finance/sc-im/Portfile

```diff
@@ -0,0 +1,27 @@
+PortSystem          1.0
+name                sc-im
+version             0.8.2
+license             BSD
+maintainers         nkorth
+
+categories          finance
+description         Spreadsheet Calculator Improvised
+long_description    An ncurses spreadsheet program for terminal
+
+platforms           darwin
+
+homepage            https://github.com/andmarti1424/sc-im
+master_sites        https://github.com/andmarti1424/sc-im/archive
+distfiles           v${version}.tar.gz
+
+checksums           rmd160  afabf063feda1800fd07e6ee13b4036c7d5a200f \
+                    sha256  7f00c98601e7f7709431fb4cbb83707c87016a3b015d48e5a7c2f018eff4b7f7 \
+                    size    1619617
+
+depends_build       bin:bison:bison
+depends_lib         port:ncurses
+
+use_configure       no
+
+worksrcdir          ${distname}/src
+destroot.destdir-append     prefix=${prefix}
```

Comment:
if you use the `makefile` PG this could become: ```makefile.prefix_name    prefix```

Reply:
That's not working for some reason - after that change I'm getting `Warning: violation by /usr` again. Any idea what I'm doing wrong?